pkg/machines/components/consoleClientResources.jsx

```diff
@@ -0,0 +1,58 @@
+/*jshint esversion: 6 */
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from "react";
+import cockpit from 'cockpit';
+
+const _ = cockpit.gettext;
+
+const ConnectWithSPICEInstructions = ({ displays }) => {
+    if (!displays.spice) {
+        return null
+    }
+    return (
+        <p>
+            {_("For better experience with SPICE, install")}&nbsp;
+            <a href='http://www.spice-space.org/download.html' target='_blank'>{_("SPICE drivers.")}</a>
+        </p>
+    )
+}
+
+const ConsoleClientResources = ({ displays }) => {
+    return (
+        <div>
+            <ConnectWithSPICEInstructions displays={displays}/>
+            <p>
+                {_("If console's window did not pop up, please make sure you have the remote-viewer application installed. To do so, either")}
+                <ul>
+                    <li>{_("use package manager:")} <i>yum/dnf install remote-viewer</i></li>
```

Comment:
The package name is "virt-viewer" in Centos, Fedora and Debian/Ubuntu, and this should add an "apt-get"